shallow_depth_insertion/Aruco[] aruco_array